app/views/delete.erb

```diff
@@ -0,0 +1,10 @@
+<% if !!@price && @price.user == session[:current_user] %>
```

Comment:
It's weird to have a view called `delete`. Delete requests generally redirect instead of render.